packages/zoe/src/zoe.js

```diff
@@ -443,7 +423,39 @@ const makeZoe = (additionalEndowments = {}) => {
         return completeOffers(instanceHandle, offerHandles);
       },
 
-      makeInvitation,
+      // Make a Zoe invite payment with an extent that is a mix of credible
+      // information from Zoe (the `handle` and `instanceHandle`) and
+      // other information defined by the smart contract. Note that the
+      // smart contract cannot override or change the values of `handle`
+      // and `instanceHandle`.
+      makeInvitation: ({
+        offerHook = () => {},
+        inviteDesc,
+        customProperties = harden({}),
+      }) => {
+        assert.typeof(
+          // TODO Once we're past the transition and all
+          // contracts have been upgraded to use inviteDesc
+          // directly, rather than as a part of customProperties,
+          // the next line can simplify to "inviteDesc,"
+          inviteDesc || customProperties.inviteDesc,
```

Comment:
What's left to do?

Reply:
Yeah, it seems like we should remove this reliance on `customProperties` in this PR

Reply:
This remains now only in the clearly "DEPRECATED" inviteAnOffer function, to be removed once we're past this transition. It is still needed atm so (I hope) the integration tests pass.